pkgs/tools/misc/ntfy-sh/default.nix

```diff
@@ -1,12 +1,15 @@
 { lib, pkgs, stdenv, buildGoModule, fetchFromGitHub, nixosTests
-, nodejs, debianutils, mkdocs, python3, python3Packages }:
+, nodejs, debianutils, mkdocs, python3, python3Packages
+, pkg-config, pixman, cairo, pango }:
 
 
 let
   nodeDependencies = (import ./node-composition.nix {
     inherit pkgs nodejs;
     inherit (stdenv.hostPlatform) system;
-  }).nodeDependencies;
+  }).nodeDependencies.override {
+    buildInputs = [ pkg-config pixman cairo pango ];
```

Comment:
`pkg-config` should be a `nativeBuildInput`, right?

Reply:
Yes, you're absolutely right, thanks for catching it! I fixed it now.